homeassistant/components/elkm1/alarm_control_panel.py

```diff
@@ -208,57 +216,68 @@ def _element_changed(self, element, changeset):
 
         if self._element.alarm_state is None:
             self._state = None
-        elif self._area_is_in_alarm_state():
+        elif self._element.alarm_state >= AlarmState.FIRE_ALARM.value:
```

Comment:
This is making the function inline. As a separate function there was a type error around `can't assign None`.